x/dex/keeper/tick.go

```diff
@@ -0,0 +1,32 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/sei-protocol/sei-chain/x/dex/types"
+)
+
+// pair -> tick size
+func (k Keeper) SetTickSizeForPair(ctx sdk.Context, pair types.Pair, ticksize sdk.Dec) {
```

Comment:
I think we should allow different tick sizes for the same pair across different contracts here just in case we need to in the future, which might involve messy migrations.